typegate/tests/utils/test.ts

```diff
@@ -349,7 +348,8 @@ export const test = ((name, fn, opts = {}): void => {
   });
 }) as TestExt;
 
-test.only = (name, fn, opts = {}) => test(name, fn, { ...opts, only: true });
+test.only = (o, fn) =>
+  test({ ...(typeof o === "string" ? { name: o } : o), only: true }, fn);
```

Comment:
I would even push to remove those to variable for an explicit option in the vanilla `test`